set_env_variables.sh

```diff
@@ -34,10 +43,10 @@ sleep 1
 ARCH=$( uname -m )
 if [ "${ARCH}" = "aarch64" ]; then
    python3 $SCRIPT_DIR/utils/setup/gen_ld_preload.py
-   LD_PRELOAD=$(<$SCRIPT_DIR/utils/setup/.ld_preload)
-   export LD_PRELOAD=$LD_PRELOAD
+   LD_PRELOAD=`cat $SCRIPT_DIR/utils/setup/.ld_preload`
```

Comment:
What's the rationale of this change?

Reply:
LD_PRELOAD=$(<$SCRIPT_DIR/utils/setup/.ld_preload) for some reason doesn't work in jenkins

LD_PRELOAD=`cat $SCRIPT_DIR/utils/setup/.ld_preload` does exactly the same as above, and additionaly I think it is a bit more clear

Reply:
Got it.

Reply:
Actually given that CI uses model_zoo_dev, instead of IS_JENKINS variable, this should be solved by passing a proper value (I assume pwd/ampere_model_library) from the level of model_zoo_dev wrapper (setup.sh -> https://github.com/AmpereComputingAI/model_zoo_dev/blob/main/setup.sh). That's the reason SCRIPT_DIR variable was here in the first place.